src/indent.cpp

```diff
@@ -430,7 +427,7 @@ static void indent_pse_pop(parse_frame_t &frm, chunk_t *pc)
    {
       if (pc != NULL)
       {
-         LOG_FMT(LINDPSE, "%4zu] (pp=%d) CLOSE [%zu,%s] on %s, started on line %d, level=%zu/%zu\n",
+         LOG_FMT(LINDPSE, "%4zu] (pp=%d) CLOSE [%zu,%s] on %s, started on line %zu, level=%zu/%zu\n",
```

Comment:
With this PR I have changed 'frm.pse[frm.pse_tos].open_line' to a member of type size_t.

Using your proposal I get an compiler error, I don't understand . Can you help?
```
/home/guy-kde/Software/uncrustify/src/indent.cpp:431:39: error: expected ‘)’ before ‘PRIuS’
          printf("started on line %30" PRIuS "\n", frm.pse[frm.pse_tos].open_line);
```